docs/source/mcmc.rst

```diff
@@ -34,12 +34,12 @@ Hamiltonian Monte Carlo
 MCMC Utilities
 --------------
 
-.. autofunction:: numpyro.hmc_util.initialize_model
+.. autofunction:: numpyro.infer.util.initialize_model
 
 .. autofunction:: numpyro.util.fori_collect
 
-.. autofunction:: numpyro.hmc_util.consensus
+.. autofunction:: numpyro.infer.hmc_util.consensus
 
-.. autofunction:: numpyro.hmc_util.parametric
+.. autofunction:: numpyro.infer.hmc_util.parametric
 
-.. autofunction:: numpyro.hmc_util.parametric_draws
+.. autofunction:: numpyro.infer.hmc_util.parametric_draws
```

Comment:
Not for this PR, but it will be nice if we had some example / tutorials to demonstrate how these are useful.

Reply:
Sure, parallel hmc is a cool feature to introduce. I'll write a tutorial when flowhmc experiment is done (which I am targeting for the next week). :)